Sources/StreamChatUI/ChatMessageList/ScrollToLatestMessageButton.swift

```diff
@@ -0,0 +1,17 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import UIKit
+
+/// A Button that is used to indicate unread messages in the Message list.
+open class ScrollToLatestMessageButton: _Button, AppearanceProvider {
+    override open func setUpAppearance() {
+        super.setUpAppearance()
+        
+        setImage(appearance.images.scrollDownArrow, for: .normal)
+        backgroundColor = appearance.colorPalette.background8
+        layer.addShadow(color: appearance.colorPalette.shadow)
+        layer.cornerRadius = 20
```

Comment:
Wonder if the corner radius shouldn't be set from the outside so you don't actually need to override the whole `setupAppearance` when you change the size of the `ScrollToLatestMessageButton` via constraints or produce some strange side-effects 🤔 I would probably set this property in `setupAppearance` inside `_ChatMessageListVC` and we are good to go :)